tasks/test-distro.js

```diff
@@ -179,15 +232,13 @@ async function verifyArchives(platforms, version) {
       // (1): verify correct contents for archive
       if (contents) {
 
-        console.log('     > extracting');
```

Comment:
As I understand we do not unzip anymore but rather parse the zipped files?

Can we keep the current unzip behavior? Rationale: I would like to integration test the built + unzipped distribution locally.

Reply:
If that is a big concern I can obviously extract myself.

Reply:
It can be done but will make the code much more complex as I believe I need to create a file system write stream then somehow pipe the read stream to there. If it's only for minor convenience purposes I'd rather not spend time on that. Any opinions?

Reply:
Also we'll just create an uncompressed file but do not need to read inside from that file as the library provides us filenames within a stream.

Reply:
If it is a huge hazzle, let's not deal with this now.